frontend/src/modules/entitieslist/components/EntitiesList.tsx

```diff
@@ -357,6 +362,9 @@ export class EntitiesListBase extends React.Component<InternalProps> {
                                     !props.batchactions.entities.length &&
                                     entity.pk === props.parameters.entity;
 
+                                const sibling = entity.sibling;
+                                const displaySiblings = entity.displaySiblings;
```

Comment:
Since you already pass the whole `entity` object as the prop to `Entity`, there's no need to also pass `displaySiblings` and `sibling` as separate props.